### PR TITLE
updated Readme.md file

### LINKED ISSUE #398: 🐛 [Bug report]: Additional dependency method
**Describe the bug**
After forking and cloning the repo the commands `npm install` and `yarn install` shows error.

**To Reproduce**
1. Fork the repo
2. Clone it
3. Run `npm install` or `yarn install`

**Expected behavior**
To resolve the issue run `npm install --legacy-peer-deps`

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser: Chrome

### PR DESCRIPTION
**Pull request by nirban256**
# Description

Updated the readme file by providing additional installation methods.

Fixes #398  (issue)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. 

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -86,6 +86,12 @@ Or
 yarn install
 ```
 
+**Note:** If the above installation didn't worked for you try running:
```
Comment (atapas) [inline README.md:89]: How about this?

**Note**: `ReactPlay` runs on React 18. However, some of our dependencies are yet to upgrade to version 18. So please use the following command when you face difficulties installing the dependencies.

```bash
npm install --legacy-peer-deps
```

**Review by atapas — COMMENTED** [review]
@nirban256 comment

File: README.md (hunk 1 above)
Comment (nirban256) [reply README.md:89]: Better than my implementation, I will change it to this then

Comment (atapas) [reply README.md:89]: Also, please use it is a quote

> **Note**: ReactPlay runs on React 18. However, some of our dependencies are yet to upgrade to version 18. So please use the following command when you face difficulties installing the dependencies.
  
  ```bash
    npm install --legacy-peer-deps
  ```

@nirban256

**Review by atapas — APPROVED** [review]

**Pull request merged**